src/components/tl/cuda/tl_cuda_team.c

```diff
@@ -230,6 +287,8 @@ ucc_status_t ucc_tl_cuda_team_create_test(ucc_base_team_t *tl_team)
                             lib->cfg.max_concurrent);
     CUDA_CHECK_GOTO(cudaStreamCreateWithFlags(&team->stream,
                     cudaStreamNonBlocking), exit_err, status);
+    CUDA_CHECK_GOTO(cudaStreamCreateWithFlags(&team->stream2,
```

Comment:
where is this used?

Reply:
>> is it possible to avoid executor if the task does not use it?
executor initialized only if collecitve needs it, collective init should set corresponding flag. For triggered post executor always initialized because it's used to satisfy input output dependency 

>>where is this used?
you are right, not needed